### ISSUE TITLE
    if to.lstrip("-+").isdigit(): AttributeError: 'int' object has no attribute 'lstrip'

### ISSUE THREAD

**Issue by Muhammad-Aadil** [Nekmo/telegram-upload#197]
* telegram-upload version: latest
* Python version: 3.9
* Operating System: Linux
* Dependencies list (run `pip freeze`):

### Description

I am trying to upload a text file on my channel, using interactive mode (telegram-upload -i) but does not work.
### What I Did

```
Traceback (most recent call last):
  File "/usr/local/bin/telegram-upload", line 8, in <module>
    sys.exit(upload_cli())
  File "/usr/local/lib/python3.10/dist-packages/telegram_upload/exceptions.py", line 64, in wrap
    return fn(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/telegram_upload/management.py", line 182, in upload
    if to.lstrip("-+").isdigit():
AttributeError: 'int' object has no attribute 'lstrip'
```

**Comment by Nekmo** [comment]
Hi, this is a bug in the 0.6.0 version. You can try to install the old version 0.5.1 meanwhile. This bug will be fixed with priority.  thanks for reporting.

```
pip install telegram-upload==0.5.1
```

**Comment by Muhammad-Aadil** [comment]
okay, Thank you ;)

**Comment by Nekmo** [comment]
Fixed in version v0.6.1. Thanks!
